appinfo/app.php

```diff
@@ -20,8 +20,9 @@
 	$l10n = $container->query('OCP\IL10N');
 	return [
 		'id' => 'githubmergetracker',
+		'order' => 50,
```

Comment:
👎 

This will spam the log:
Undefined index: order at /var/www/html/lib/private/legacy/app.php(511) : runtime-created function#1